src/tests/dbus-tests/test_btrfs.py

```diff
@@ -399,3 +399,10 @@ def test_get_default_subvolume_id(self):
             default_id = dev.obj.GetDefaultSubvolumeID(self.no_options,
                              dbus_interface=self.iface_prefix + '.Filesystem.BTRFS')
             self.assertEqual(default_id, 5)
+
+            dev.obj.SetDefaultSubvolumeID(5, self.no_options,
```

Comment:
This needs to be `dbus.UInt32(5)`, `5` is treated as int32 by Python DBus.

Reply:
Thanks, fixed!